docs/playground/README.md

```diff
@@ -0,0 +1,90 @@
+# Plutus Playgrounds
+
+*Plutus Playgrounds* is a lightweight, easy-to-use, web-based environment for exploratory Plutus development. It facilitates the development and execution of Plutus contracts without the overhead of installing and maintaining a full development environment and blockchain testnet. Nevertheless, it is built around the original Plutus platform components, including the use of Plutus Core for on-chain contract code, and supports the full Plutus language. Hence, contracts developed with Plutus Playgrounds can eventually be deployed to the Cardano blockchain.
```

Comment:
"the full Plutus language"? Is that even a thing? ;)

Reply:
It is a way of saying, "Plutus as we expect it to eventually run on the real Cardano blockchain", but with less words 😅